lib/eldap/src/eldap.erl

```diff
@@ -36,6 +36,19 @@ Type definitions that are used more than once in this module:
 - **`referrals() =`** - `[Address = string()]` The contents of `Address` is
   server dependent.
 """.
+
+-type search_option() :: {base, string()} | {filter, filter()} |
+                         {scope, scope()} | {attributes, [string()]} |
+                         {deref, dereference()} | {types_only, boolean()} | {timeout, integer()}.
+-type handle() :: term().
+-type modify_op() :: term().
+-type scope() :: term().
+-type dereference() :: term().
+-type filter() :: term().
+-type referrals() :: [string()].
```

Comment:
You need to add a doc for this type.
It used to be "[Address = string()] The contents of Address is server dependent"